examples/oapp/layerzero.config.ts

```diff
@@ -21,6 +21,9 @@ const config: OAppOmniGraphHardhat = {
     contracts: [
         {
             contract: fujiContract,
+            config: {
+                callerBpsCap: BigInt(1000),
```

Comment:
Per discussions w/ Riley, I would recommend you set a default of `300`.

Reply:
Updated!